zc_common/remote_resource/relations.py

```diff
@@ -6,23 +6,19 @@
 from rest_framework_json_api.relations import ResourceRelatedField
 
 from zc_common.remote_resource.models import RemoteResource
+from zc_common.remote_resource.request import get_route_from_fk
 
 
 class RemoteResourceField(ResourceRelatedField):
 
-    def __init__(self, related_resource_path=None, **kwargs):
```

Comment:
I would suggest to not change the behavior of this Model for now. Lets put back all the changes.